.pre-commit-config.yaml

```diff
@@ -7,16 +7,15 @@ repos:
         entry: poetry run black
         language: system
         minimum_pre_commit_version: 2.9.2
-        require_serial: true
         types_or: [ python, pyi ]
+        exclude: '^src/puya/(_vendor|_typeshed)/'
```

Comment:
can the exclude redefinition be removed now that require_serial is false?

Reply:
yes it can